products.js

```diff
@@ -282,25 +480,37 @@ class DiffStats {
           expectedText = expectedDom.head.firstChild.innerHTML;
           gotText = this.tuningRoutine(nodeToCssMap, ...coeffs)(sourceDom).map(fnode => fnode.element.innerHTML)[0];
         } else if (this.feature === 'price') {
-          //strip whitespace and dollar sign if there is one when comparing price
+          //strip whitespace, dollar sign, trailing zeros if when comparing price
 
-          // console.log(expectedDom.body.firstChild.outerHTML);
           // expectedText = expectedDom.body.firstChild.outerHTML;
           // gotText = this.tuningRoutine(nodeToCssMap)(sourceDom).map(fnode => fnode.element.outerHTML)[0];
-          expectedText = expectedDom.body.firstChild.textContent.replace('$', '').trim();
-          gotText = this.tuningRoutine(nodeToCssMap, ...coeffs)(sourceDom).map(fnode => fnode.element.textContent.replace('$', '').trim())[0];
 
-        }
+          expectedText = expectedDom.body.firstChild.textContent.trim().replace('$', '').replace(/\s/g,'').replace(/[^0-9$.-]/g, '').replace(/(\.[0-9]*?)0+$/, '');
```

Comment:
Ah, I didn't see the `^` in my haste. But `'59.7700'.replace(/(\.[0-9]*?)0+$/, '');` is `59`. You didn't mean to strip off the 77¢, did you?